src/metabase/driver/sql/query_processor.clj

```diff
@@ -63,7 +63,7 @@
                          (.getCanonicalName (class params)))
                     {:type  qp.error-type/invalid-query
                      :query params})))
-  (case hx/*honey-sql-version*
+  (case (long hx/*honey-sql-version*)
```

Comment:
Not strictly related but now that #28205 is in, it's complaining about all these things. This shuts the compiler up.